app/infrastructure/database/repositories.py

```
@@ -1,135 +1,91 @@
-
-    @abstractmethod
-    async def update_token_usage(self, user_id: str, tokens_used: int) -> None:
-        pass
-
-    @abstractmethod
-    async def create_user(self, user_data: dict) -> User:
-        pass
 
+class UserRepositoryHelper:
+    
+    def __init__(self, repo=TortoiseUserStore()):
```

Ruff (0.12.2)

Do not perform function call `TortoiseUserStore` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)